torchcfm/optimal_transport.py

```diff
@@ -37,6 +37,21 @@ def __init__(
         self.kwargs = kwargs
 
     def get_map(self, x0, x1):
+        """Compute the OT plan (wrt squared Euclidean cost) between a source and a target
+        minibatch.
+
+        Parameters
+        ----------
+        x0 : Tensor, shape (bs, *dim)
```

Comment:
This is good, but FYI you can now do things like `xo: torch.Tensor` in the signature itself to have typing both be documented and checked by python automatically.

Reply:
That's in addition of the doc right? Because it does not tell you the expected dimension, only the required type.